src/kaggle/api/kaggle_api_extended.py

```diff
@@ -93,8 +92,9 @@ def __init__(self, fullpath, format):
     def __enter__(self):
         self._temp_dir = tempfile.mkdtemp()
         _, dir_name = os.path.split(self._fullpath)
-        self.path = shutil.make_archive(os.path.join(self._temp_dir, dir_name),
-                                        self._format, self._fullpath)
+        self.path = shutil.make_archive(

```

Comment:
Yes, I'm running it in the container. I pretty much have to if I want to work on the C# and Python together. I tried opening the project in IntelliJ (because the Python plugin for Rider is currently not operational), but quickly realized I'd have to go through and install all the dependencies again. I didn't want to do that, since I don't know how many are pre-installed in the container. I mention that because I thought that's where the reformatting occurred (but `yapf` is more likely). I did restore almost all the format changes.

Reply:
In the `Development` section, I'm adding this text. Let me know if there's any problem with it:

### Kaggle Internal

Obviously, this depends on Kaggle services. When you're extending the API and modifying
or adding to those services, you should be working in your Kaggle mid-tier development
environment. You'll run Kaggle locally, in the container, and test the Python code by
running it in the container so it can connect to your local testing environment.

Reply:
That's the `Development` section of `README.md`.

Reply:
In case you are not aware, the releases for `kagglehub` and `kaggle-api` are done using Google Cloud Build. Anyone can make a release in a consistent environment. You just need to trigger Cloud Build with a CLI command.

Docmentation for `kagglehub`: https://g3doc.corp.google.com/cloud/kaggle/models/g3doc/kagglehub.md#step-2-create-a-new-release

For `kaggle-api`,  it is documented here: https://g3doc.corp.google.com/company/teams/kaggle/cli/index.md#release